pkg/apis/serving/v1beta1/predictor.go

```diff
@@ -44,6 +44,8 @@ type PredictorSpec struct {
 	Triton *TritonSpec `json:"triton,omitempty"`
 	// Spec for ONNX runtime (https://github.com/microsoft/onnxruntime)
 	ONNX *ONNXRuntimeSpec `json:"onnx,omitempty"`
+	// Spec for HuggingFace runtime (https://github.com/huggingface)
+	HuggingFace *HuggingFaceRuntimeSpec `json:"huggingface,omitempty"`
```

Comment:
Actually we are trying to deprecate these individually defined runtime types as we are moving users to the model spec instead. @sivanantha321 has a PR for this https://github.com/kserve/kserve/pull/3144

Reply:
Yeah I am aware of that. IMO we should include this for consistency (better UX) and deprecate all the individually defined runtimes together as part of https://github.com/kserve/kserve/pull/3144.